packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -392,25 +383,23 @@ class Checkout extends Component<
                     />
                 }
             >
-                <LazyContainer loadingSkeleton={<CustomerSkeleton />}>
-                    <Customer
-                        checkEmbeddedSupport={this.checkEmbeddedSupport}
-                        isEmbedded={isEmbedded()}
-                        isSubscribed={isSubscribed}
-                        isWalletButtonsOnTop = {isWalletButtonsOnTop}
-                        onAccountCreated={this.navigateToNextIncompleteStep}
-                        onChangeViewType={this.setCustomerViewType}
-                        onContinueAsGuest={this.navigateToNextIncompleteStep}
-                        onContinueAsGuestError={this.handleError}
-                        onReady={this.handleReady}
-                        onSignIn={this.navigateToNextIncompleteStep}
-                        onSignInError={this.handleError}
-                        onSubscribeToNewsletter={this.handleNewsletterSubscription}
-                        onUnhandledError={this.handleUnhandledError}
-                        step={step}
-                        viewType={customerViewType}
-                    />
-                </LazyContainer>
+                <Customer
+                    checkEmbeddedSupport={this.checkEmbeddedSupport}
+                    isEmbedded={isEmbedded()}
+                    isSubscribed={isSubscribed}
+                    isWalletButtonsOnTop = {isWalletButtonsOnTop}
+                    onAccountCreated={this.navigateToNextIncompleteStep}
+                    onChangeViewType={this.setCustomerViewType}
+                    onContinueAsGuest={this.navigateToNextIncompleteStep}
+                    onContinueAsGuestError={this.handleError}
+                    onReady={this.handleReady}
+                    onSignIn={this.navigateToNextIncompleteStep}
+                    onSignInError={this.handleError}
+                    onSubscribeToNewsletter={this.handleNewsletterSubscription}
+                    onUnhandledError={this.handleUnhandledError}
+                    step={step}
+                    viewType={customerViewType}
+                />
```

Comment:
How is the experience with the old wallet button list after the skeleton removal? Is there anything we need to adjust?